database/dml/dml_test.go

```diff
@@ -22,6 +22,8 @@ func TestDML_NewMap_Postgres(t *testing.T) {
 				"all":         postgres.ListBuilds,
 				"repo":        postgres.ListRepoBuilds,
 				"repoByEvent": postgres.ListRepoBuildsByEvent,
+				"org":         postgres.SelectBuildByOrg,
```

Comment:
Does `"org": postgres.SelectBuildByOrg` do the same as `"repo": postgres.ListRepoBuilds`?

if so should the names be similar?

Reply:
Renamed to ListOrgBuilds.